frontend/public/locales/en/status.json

```diff
@@ -4,6 +4,7 @@
   "status-checker-content": "The Status Checker is a tool that lets you check the status of your Canadian Dental Care Plan application.",
   "online-status-heading": "Who can check their application status online?",
   "online-status-content": "If you applied for the Canadian Dental Care Plan, you can check your status online.",
+  "terms-conditions": "En utilisant le vérificateur de l'état d'une demande, vous acceptez les conditions générales d'utilisation et à l'énoncé de confidentialité.",
```

Comment:
The French text is in the `/en/status.json` while the English text is in `/fr/status.json`